src/lib/operators/validate.cpp

```diff
@@ -65,8 +65,10 @@ std::shared_ptr<const Table> Validate::_on_execute(std::shared_ptr<TransactionCo
 
     // If the columns in this chunk reference a column, build a poslist for a reference column.
     if (ref_col_in) {
-      DebugAssert(chunk_in.references_only_one_table(),
-                  "Input to Validate contains a Chunk referencing more than one table.");
+      if (!chunk_in.references_only_one_table()) {
```

Comment:
Why did you replace the assertion with this? This looks like it can lead to wrong results.